packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -294,10 +304,39 @@ class CoresPerSocket extends Component {
 export default class TabAdvanced extends Component {
   componentDidMount () {
     getVmsHaValues().then(vmsHaValues => this.setState({ vmsHaValues }))
+    const { cpuMask } = this.props.vm
+    this.setState({ vmCpuMask: cpuMask && cpuMask.split(',') })
```

Comment:
The CPU mask is an advanced setting so I think we should keep it as a raw string.